recipes/minimock/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "MiniMock" %}
+{% set version = "1.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sarnold/{{ name|lower }}/archive/{{ version }}.tar.gz
```

Comment:
Could you use the official repo and add your changes as a patch? Have you asked upstream about letting you have merge rights on their repo?

Reply:
Upstream is 5 years cold, and I think that's long enough to declare myself "upstream maintainer".  If/when the original author returns I'll be happy to make a PR, but until then I'm all there is.

Reply:
Looks like upstream was active on github yesterday. Can you open an issue on https://github.com/lowks/minimock/issues?

Reply:
I can make a PR but the last activity was "Commits on Feb 14, 2016".  What are you seeing that I'm not seeing?

Reply:
PR pushed.

Reply:
Hmm, still seeing upstream activity in another repo, but nothing on the minimock PR...  I shall ping upstream again...

Reply:
Thanks. From what I see from your PR, apart from CI, code formatting, the actual changes needed is a small fix right? Can we just use the upstream repo and apply your PR on top as a patch?

Reply:
Is there a limit on patch size?  (just checking)

Reply:
> Is there a limit on patch size? (just checking)

Nope.

Reply:
aaand we got some attention upstream...

Reply:
PR is merged, new release on github, nothing on pypi yet.  Can we go ahead with upstream github for now?  I can update again whenever pypi gets updated.

Reply:
>  Can we go ahead with upstream github for now? I can update again whenever pypi gets updated.

Sure.

Reply:
okay! :+1: